schema.json

```diff
@@ -1108,70 +1148,113 @@
     "name": "Products",
     "settings": [
       {
-        "type": "checkbox",
-        "label": "Show Quickview",
-        "force_reload": true,
-        "id": "show_product_quick_view"
+        "type": "heading",
+        "content": "Product sale badges"
       },
       {
-        "type": "heading",
-        "content": "Product pages"
+        "type": "select",
+        "label": "Show product sale badges",
+        "id": "product_sale_badges",
+        "force_reload": true,
+        "options": [
+          {
+            "value": "none",
+            "label": "None"
+          },
+          {
+            "value": "topleft",
+            "label": "Top Left"
+          },
+          {
+            "value": "sash",
+            "label": "Diagonal"
+          },
+          {
+            "value": "burst",
+            "label": "Burst"
+          }
+        ]
       },
       {
         "type": "text",
-        "label": "Product sale price label",
-        "id": "pdp-sale-price-label"
+        "label": "Product price label (sale)",
+        "force_reload": true,
```

Comment:
@pedelman @kchu93 thank you for doing this as part of this PR.

Reply:
👍

Reply:
@karen-white @bookernath we will need to get all our theme developers to add `"force_reload": true` for all fields of `"type": "text"`